smoke-tests/smoke-sdk-grpc.bats

```diff
@@ -3,36 +3,46 @@
 load test_helpers/utilities
 
 CONTAINER_NAME="app-sdk-grpc"
+COLLECTOR_NAME="collector"
 OTEL_SERVICE_NAME="otel-python-example"
+TRACER_NAME="hello_world_tracer"
+METER_NAME="hello_world_meter"
 
 setup_file() {
 	echo "# 🚧" >&3
-	docker-compose up --build --detach collector ${CONTAINER_NAME}
-	wait_for_ready_app ${CONTAINER_NAME}
-	curl --silent "http://localhost:5000"
+	docker-compose up --build --detach collector
+	wait_for_ready_collector ${COLLECTOR_NAME}
```

Comment:
This differs from most of our other smoke tests because this app just runs and exits - it doesn't have an endpoint. This ensures the collector is ready before the app runs and sends telemetry.